debug/testlib.py

```diff
@@ -1210,6 +1210,11 @@ def disable_pmp(self):
             self.gdb.p("$pmpcfg0=0xf")  # TOR, R, W, X
             self.gdb.p("$pmpaddr0=0x%x" %
                     ((self.hart.ram + self.hart.ram_size) >> 2))
+            # If TOR is unsupported (detected via WARL) then use NAPOT instead
+            self.gdb.command("flush regs")
```

Comment:
Do you need this? I thought the CSRs were write-through.

Reply:
Was added out of caution, but you're right, wholly unnecessary